### DISCUSSION TITLE
台服需要开加速器吗，为什么一直是这个界面?一直applying builds。

### DISCUSSION THREAD

**Discussion by AdrienL999** [cangzhang/ChampR#287] [General]
<img width="452" alt="屏幕截图 2023-07-16 130905" src="https://github.com/cangzhang/champ-r/assets/139670140/246b189b-cb7d-464e-8b2b-c9e04af61dd3">

**Comment by cangzhang** [comment]
那是因为你点了火箭按钮（导入装备）
右边那个下拉菜单是选择天赋装备源，进入游戏 ban pick 阶段时会根据你选择的数据源拉取数据

**Reply by AdrienL999** [reply]
好的，谢谢！

**Comment by cangzhang** [comment]
台服导入出装不生效的话可以用这里的版本：
https://github.com/cangzhang/champ-r/actions/runs/5566797314
